.github/actions/check-image-tags-exist/action.yml

```diff
@@ -49,4 +43,4 @@ runs:
       shell: bash
       run: |
         echo "last_commit_tag_exists: ${{ steps.last_commit_image_exists.outputs.last_commit_tag_exists }}"
-        echo "common_ancestor_commit_tag_exists: ${{ steps.ancestor_commit_image_exists.outputs.common_ancestor_commit_tag_exists }}"
\ No newline at end of file
+        echo "common_ancestor_commit_tag_exists: ${{ steps.ancestor_commit_image_exists.outputs.common_ancestor_commit_tag_exists }}"
```

Comment:
Why do we still need to echo `common_ancestor_commit_tag_exists`? in fact we could remove the above step `ancestor_commit_image_exists`, right?